src/core/glue-defaults.ts

```diff
@@ -10,7 +10,7 @@ export class GlueDefaults {
       maxRetries: 1,
       timeout: cdk.Duration.hours(10),
       securityConfiguration: new glue.SecurityConfiguration(scope, id, {
-        securityConfigurationName: `${id}-security-config`,
```

Comment:
Do we need to provide a name for the security configuration?

Reply:
I don’t recall why I needed the name in there. I’ll validate generation works with our integration test

Reply:
Yea this unnecessary, taking out